.github/actions/openziti-tunnel-build-action/ubuntu-20.04/Dockerfile

```diff
@@ -28,9 +34,16 @@ RUN apt-get -y update \
         libssl-dev \
     && rm -rf /var/lib/apt/lists/*
 
-RUN curl -L https://cmake.org/files/v${CMAKE_VERSION%.*}/cmake-${CMAKE_VERSION}-linux-x86_64.sh -o cmake.sh \
+RUN dpkg --add-architecture arm64 && dpkg --add-architecture armhf
```

Comment:
is this why you switched to Debian?

Reply:
Yes. You can add foreign arch in Ubuntu, but `apt update` breaks immediately, and adding a new repo sources list for Ubuntu ports doesn't unbreak `apt update`.